simplq/src/services/axios.js

```diff
@@ -14,6 +14,8 @@ const handleApiErrors = (err) => {
         `There's a problem with the data you've entered ${err.response.data.message}`
       )
     );
+  } else if (err.response.status === 401) {
+    window.location.href = `${window.location.origin}/unauthorized`;
   } else {
```

Comment:
@daltonfury42 what do you think about this approach? Should we go ahead with this or conditionally switch to the unauthorized page from within the admin component?

Reply:
I actually don't know. I am trying to think of what could go wrong in our current implementation if we go this route. 🤔 

If we want to conditionally switch to unauthorised page from admin component, we need to let admin component know that it was 401. But we took the call to handle API errors at one place in here, so it might be difficult...(It's a design limitation). Do you see a way?

If not, can we go ahead with this, and then see if it doesn't cause problems? (I can't see any right now, though it might give issues later)

Reply:
Okay let's do this for now and change later if needed.